diagnostic.md

```diff
@@ -21,13 +21,18 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com
```

Comment:
So close! You actually need `/donuts` at the end, like this: `curl --include --request GET http://www.example.com/donuts`